src/funcs.c

```diff
@@ -57,6 +57,39 @@
 
 #include <src/hookintrprtr.h>
 
+static ModuleStateOffset funcsStateOffset;
+typedef struct {
+    Int RecursionDepth;
+    Obj ExecState;
+} funcsModuleState;
```

Comment:
Here's one last nitpick: Virtually every other typedef or global variable starts with an uppercase letter. Why not these two as well? I.e. change `funcs` to `Funcs` here and elsewhere?

Reply:
Anyway, I'll merge this as-is, and we can change the case in one of the next commits.